lib/PillInput/PillInput/PillInput.js

```diff
@@ -34,6 +35,11 @@ const PillInput = ({
     setPills(newPills);
   };
 
+  const removeLastPill = () => {
+    const newPills = pills.length > 1 ? pills.slice(0, pills.length - 1) : [];
```

Comment:
I'm not too up on how slice works, but if `pills.length` = 1, wouldn't slice still return `[]`? Meaning this ternary might not be needed?

Reply:
100% 🙏 

I added this when I had another bug thinking it was a fix - then forgot to remove it 👍